setup.py

```diff
@@ -31,4 +31,5 @@ def read(fname):
         "oemof.tabular @ git+https://git@github.com/oemof/oemof-tabular@dev#egg=oemof.tabular",
         "frictionless",
     ],
+    extras_require={"dev": ["pytest", "black==20.8b1", "coverage", "flake8"]},
```

Comment:
Does it have to be this specific black version?

Reply:
I've orientated also here on https://github.com/rl-institut/oemof-B3/pull/25/files. I remember having checks that didn't pass using a different black version in the past but I don't know which versions can actually be used.

Reply:
Ok, let's leave it like this. In general, it would be good to have a comment here why it is this specific version. A constraint on the version should have a reason, otherwise you restrict yourself from getting new versions with potentially useful fixes and features without knowing why.

Reply:
True! I created an issue https://github.com/rl-institut/oemoflex/issues/24, where this topic can be examined.